### ISSUE TITLE
Proxy voting feature

### ISSUE THREAD

**Issue by enajx** [loomio/loomio#9380]
Is there a plan of implementing a proxy voting option —as in delegative voting in liquid democracy?

**Comment by robguthrie** [comment]
I'm happy that the architecture is suitable to support doing this, but we're waiting until a significant paying customer can show it's a priority for them.

**Comment by nyabinary** [comment]
Any reason why this was cloosed instead of staying open? :3

**Comment by robguthrie** [comment]
I think that the two issues you linked are quite different. 
One is a user delegating their vote to someone else - ie user takes action,
The other is a facilitator/admin voting on behalf of the user. ie admin takes action.

**Comment by robguthrie** [comment]
Anyway, do you have a real use case for delegated voting, within a real organization?

**Comment by Lord-Valen** [comment]
> I think that the two issues you linked are quite different. One is a user delegating their vote to someone else - ie user takes action, The other is a facilitator/admin voting on behalf of the user. ie admin takes action.

Yes, it seems that the other issue is concerning proxy voting, while this is concerning delegated voting. The title should probably be changed as it becomes easy to conflate the two issues.

**Comment by Lord-Valen** [comment]
> Anyway, do you have a real use case for delegated voting, within a real organization?

The Nix community is currently considering the possibility of using both liquid democracy and Loomio, respectively, for use in the near future. You can see the developing conversations [here](https://nixpkgs.zulipchat.com/#narrow/stream/435724-governance/topic/Liquid.20Democracy) and [here](https://nixpkgs.zulipchat.com/#narrow/stream/435724-governance/topic/Loomio).

**Comment by robguthrie** [comment]
Oh that's so cool to learn of, thanks for sharing. We'd be happy to support.

**Comment by robguthrie** [comment]
Nothing actionable here
